transports/quic/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 <!-- Update to libp2p-core v0.43.0 -->
 
+- Deprecate `Config::support_draft_29`.
+  See [PR 5786](https://github.com/libp2p/rust-libp2p/pull/5786).
+
```

Comment:
`libp2p-quic-v0.12.0` is already released. You need to bump the version of it to `0.12.1` and move this to a new section above.

Reply:
I mean the version of `libp2p-quic` itself also needs to be bumped :)